packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -25,19 +25,22 @@ class FrameUtil {
    * @param {ReactNode} target - The select wrapper.
    * @param {ReactNode} dropdown - The dropdown.
    * @param {number} maxHeight - The maxHeight of the dropdown dropdown.
+   * @param {Object} currentState - The current component state.
    * @param {boolean} useSemanticDropdown - If the dropdown should be rendered semantically instead of in a portal
    * @return {Object} - The calculated dropdown attributes.
    */
-  static dropdownPosition(props, target, dropdown, maxHeight, useSemanticDropdown = false) {
+  static dropdownPosition(props, target, dropdown, maxHeight, currentState, useSemanticDropdown = false) {
     const style = ({ ...props }).style || {};
     const { height } = dropdown.getBoundingClientRect();
     const { bottom, width, top } = target.getBoundingClientRect();
 
     const spaceBelow = window.innerHeight - bottom;
     const maximumHeight = parseInt(style.maxHeight || maxHeight, 10) || Infinity;
-    const canFitBelow = maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top;
+    let canFitBelow = (maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top);
+    if (useSemanticDropdown && currentState !== undefined && currentState.resizeOnSearch && currentState.isAbove && canFitBelow) {
```

Comment:
```jsx
if (useSemanticDropdown && currentState !== undefined && currentState.resizeOnSearch && currentState.isAbove && canFitBelow)
```
I think this if condition is added to prevent dropdownlist from displaying below form-select on search ( when dropdownlist has only one item ) . if this is the reason for having this condition then instead of using currentState and other variables, Can we modify the existing condition to like below to ensure dropdown will Fit below only when it has space to accommodate two or more options. 
```jsx
const canFitBelow = (maximumHeight < spaceBelow || ( height * 2 ) < spaceBelow || spaceBelow > top);
```

Reply:
This condition is added so that if the dropdown has initially opened above it again opens above the select after search, that is why isAbove is also added

Reply:
disregard my suggestion of having `height * 2` in condition as it doesn't serve the purpose of this condition. but with IsAbove being added as condition regardless of space available it will always display dropdown above the select like shown below.

![form-select-detach](https://user-images.githubusercontent.com/12869809/78811563-acb26480-79e7-11ea-8197-1498938b9e8d.gif)

Is it expected  to display above form-select on search even though it has plenty of space below the form select...? 🤔

Reply:
Fixed a2c35f5